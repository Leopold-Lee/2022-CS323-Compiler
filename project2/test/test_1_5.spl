int main(
{
    int a = 0x0123
    b = 0x321;
    float c = -2134.1;
    if(a > 0
    {
        a = 0;
    }
}