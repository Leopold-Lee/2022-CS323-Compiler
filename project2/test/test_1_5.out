Error type B at Line 1: Missing closing parenthesis ')'
Error type A at Line 3: unknown lexeme 0x0123
Error type B at Line 3: Missing semicolon ';'
Error type B at Line 4: Missing specifier
Error type B at Line 6: Missing closing parenthesis ')'
